www/js/con-author.js

```diff
@@ -745,9 +745,6 @@ define([
 				m = new memory({data: units});
 				unitsWidget.set("store", m);
```

Comment:
```
var m = new memory({data: descriptions});
descriptionWidget.set("store", m);
m = new memory({data: units});
unitsWidget.set("store", m);
```
It looks like m isn't ever being re-used.  Could we just re-write this to:

```
descriptionWidget.set("store", new memory({data: descriptions}));
unitsWidget.set("store", new memory({data: units}));
```
?